Functions/Misc/colors

```diff
@@ -82,10 +82,13 @@ for k in ${(k)color}; do color[${color[$k]}]=$k; done
 for k in ${color[(I)3?]}; do color[fg-${color[$k]}]=$k; done
 
 # This is inaccurate, but the prompt theme system needs it.
+# Note: "gray" is US spelling.
```

Comment:
The prompt system only relies on `grey`, not on `gray`, but I think line 84 does _not_ need to be changed, since someone might write down the road a prompt theme that relies on `gray`.